src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -19,6 +19,11 @@
  */
 public abstract class AbstractWindowBuilding<B extends AbstractBuildingHut.View> extends AbstractWindowSkeleton
 {
+    /**
+     * Id of the type label in the GUI.
+     */
+    private static final String LABEL_BUILDINGTYPE = "type";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LABEL_BUILDINGTYPE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1068)